### ISSUE TITLE
SIMD subgroup meeting on 2022-04-15

### ISSUE THREAD

**Issue by ngzhian** [WebAssembly/relaxed-simd#68]
This is a joint meeting with flexible-vectors and any SIMD related presentations are also welcome, e.g. new architectures or toolchains, longer term discussion that don't fit into flexible-vectors or relaxed-simd.

The meeting will be on Friday, April 15, at 9:00AM - 10:00AM PDT/ 5:00PM - 6:00PM CET.

If this meeting doesn't already appear on your calendar, or you are a new attendee, please fill out this [form](https://forms.gle/9eB2ZYaziPEcTJabA) to attend.

**Comment by ngzhian** [comment]
@penzn fyi

**Comment by ngzhian** [comment]
At today's CG meeting, we presented updates on the Relaxed SIMD proposal, generated very good discussion w.r.t. spec text and non-determinism (will link meeting notes here when they are up), and **successfully polled for Phase 3**. Congratulations and thanks everyone for your work! 🎉 

I am OOO this Friday and will not make this meeting, if anyone who attended today's CG meeting would like to volunteer to run discussions about the CG's feedback, please do, otherwise I can do that next SIMD subgroup meeting.

**Comment by dtig** [comment]
Thanks @ngzhian for presenting today, and steering the proposal to a successful phase 3 vote! I'm happy to help with the discussion at the SIMD subgroup meeting this week if needed, though my preference would be to start this discussion on an issue first, and bring it up for discussion at a future meeting. This is mostly so we have a place to gather all the feedback, especially if there are users that are expecting implementation dependent behavior that may not usually attend the SIMD meetings.

**Comment by ngzhian** [comment]
Thanks @dtig , i'll wait for the meeting notes before posting an issue on it.

**Comment by dtig** [comment]
Good point, added a PR [here](https://github.com/WebAssembly/meetings/pull/1011) for the meeting notes.

**Comment by ngzhian** [comment]
I propose we cancel the meeting tomorrow, I filed #71 , we can have some discussion there first, then bring it to a meeting 2 weeks from now.

**Comment by penzn** [comment]
I'd like to revisit the feature detection discussion if @tlively is going to be there, but we can also do it offline.

**Comment by tlively** [comment]
The short story on feature detection from my side is that I have no plans to do any further work on it for now. I'd be happy to discuss further at a meeting or offline. Unless anyone thinks it's urgent or if we otherwise decide to meet tomorrow, let's push the discussion to the next meeting.

**Comment by ngzhian** [comment]
Filed #72 for the next meeting.
